### PR TITLE
Binary support for Redis

### LINKED ISSUE #96: Add Redis service as binary

### PR DESCRIPTION
**Pull request by biswaroop1547**
fixes #96

### CODE REVIEW

**Comment by casperdcl** [comment]
- [x] depends on https://github.com/premAI-io/prem-services/pull/131
- [ ] should this PR target `dev` or `v1`?

File: vector-store-redis-binary/manifest.json
Comment (casperdcl) [inline vector-store-redis-binary/manifest.json:1]: prob missing `"serviceType": "binary"`?

Comment (biswaroop1547) [reply vector-store-redis-binary/manifest.json:1]: 😭

File: vector-store-redis-binary/manifest.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        "vector-store"
+    ],
+    "icon": "",
+    "modelInfo": {},
+    "defaultPort": 6380,
+    "defaultExternalPort": 6380,
+    "binariesUrl": {
+        "aarch64-apple-darwin": null,
+        "x86_64-apple-darwin": null,
+        "universal-apple-darwin": "https://raw.githubusercontent.com/premAI-io/prem-services/main/vec-redis/setup-redis.sh"
```
Comment (casperdcl) [inline vector-store-redis-binary/manifest.json:18]: Depends on https://github.com/premAI-io/prem-services/pull/131 being merged.

Or for testing pre-merge:

```suggestion
        "universal-apple-darwin": "https://raw.githubusercontent.com/biswaroop1547/prem-services/redis-binary/vec-redis/setup-redis.sh"
```

**Review by casperdcl — APPROVED** [review]

File: chat-mistral-7b-instruct/manifest.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "interfaces": [
         "chat"
     ],
-    "defaultPort": 8000,
+    "defaultPort": 8447,
```
Comment (tiero) [inline chat-mistral-7b-instruct/manifest.json:16]: Why this change is needed here?

**Comment by tiero** [comment]
You started from wrong history of commits.
Please open a PR against dev with only redis folder to be added, no any other change to others services

File: chat-mistral-7b-instruct/manifest.json (hunk 2 above)
Comment (tiero) [reply chat-mistral-7b-instruct/manifest.json:16]: ok i think this and other services commits should not be merged together

**Comment by casperdcl** [comment]
yeah this PR probably needs rebasing :)
